docs/docs/authenticate/index.md

```diff
@@ -54,34 +69,32 @@ default['firezone']['authentication']['oidc'] = {
 }
 ```
 
-1. `discovery_document_uri`: This URL returns a JSON with information to
-construct a request to the OpenID server.
-1. `client_id`: The client ID of the application.
-1. `client_secret`: The client secret of the application.
-1. `redirect_uri`: Instructs OIDC provider where to redirect after authentication.
-This should be your Firezone `EXTERNAL_URL + /auth/oidc/<provider_key>/callback/`
-(e.g. `https://firezone.example.com/auth/oidc/google/callback/`).
-1. `response_type`: Set to `code`.
-1. `scope`: [OIDC scopes](https://openid.net/specs/openid-connect-basic-1_0.html#Scopes)
-to obtain from your OIDC provider. This should be set to `openid email profile`
-or `openid email profile offline_access` depending on the provider.
-1. `label`: The button label text that shows up on your Firezone login screen.
-
 We've included instructions on how to set up Firezone with several popular
 identity providers:
 
-* [Azure AD]({%link docs/authenticate/azure-ad.md%})
-* [Google]({%link docs/authenticate/google.md%})
-* [Okta]({%link docs/authenticate/okta.md%})
-* [Local email/password authentication (default)]({%link docs/authenticate/web-auth.md%})
+<!-- markdownlint-disable MD013 -->
+<!-- markdownlint-disable MD034 -->
+
+| OIDC provider | offline_access | Link to Guide                                        |
```

Comment:
Kinda confused about this table -- is it a feature matrix? Should the `offline_access` header be renamed to like `User Synchronization` or something, and then the fields updated to be like

> 'offline_access' required in `scope` parameter